src/components/forms/pattern-form/pattern-form.js

```diff
@@ -123,14 +125,25 @@ const PatternForm = ({ pattern, id, isEdit }) => {
       const newPattern = createPattern(values);
 
       if (isEdit && upload instanceof File) {
```

Comment:
You don't check if `uploadConstructorImg` is  instance of `File`

Reply:
fixed